esp-hal-common/src/embassy/time_driver_systimer.rs

```diff
@@ -96,11 +96,11 @@ impl EmbassyTimer {
         })
     }
 
-    fn clear_interrupt(&self, id: usize) {
+    fn interrupt_clear(&self, id: usize) {
```

Comment:
This feels a bit wrong to me among all the other `<verb>_<noun>` method names (`trigger_alarm`, `set_alarm` and even `on_interrupt` though that's a bit of a stretch as `on` isn't a verb). Is the goal to bring this in line with `interrupt_enable`? Risking a bit of bike shedding, but wouldn't it be better then to change `interrupt_enable` to `enable_interrupt`?

Reply:
I agree, imo `<verb>_<noun>` is much easier for me to understand.